documentation/src/main/wiki/HttpServiceAdapter.md

```diff
@@ -111,6 +119,19 @@ Example configuration of a [[Service Knot|ServiceKnot]]:
         "params": {
           "path": "/service/twitter/user/{header.userId}"
         }
+      },
+      {
+        "name" : "javabooks",
```

Comment:
Good example but can I display those books with any example page? If yes please provide here a link to this page with information how to run demo.